scripts/concurrent_task_utils.py

```diff
@@ -58,14 +99,28 @@ def __init__(self, func, verbose, semaphore, name):
 
     def run(self):
         try:
-            self.output = self.func()
+            self.task_results = self.func()
             if self.verbose:
-                log('LOG %s:' % self.name, show_time=True)
-                log(self.output)
-                log('----------------------------------------')
+                for task_result in self.task_results:
+                    # The following section will print the output of the lint
+                    # checks.
+                    if task_result.name:
```

Comment:
Instead of relying on this field to decide how to print the message, create another filed report_enabled (with True default value of keyword param) and disable it for the backed tests. (Let me know if you need more info on this!)

Reply:
Done.